wagtail/images/static_src/wagtailimages/js/focal-point-chooser.js

```diff
@@ -1,6 +1,13 @@
 var jcropapi;
 
 function setupJcrop(image, original, focalPointOriginal, fields) {
+  var $chooser = image.closest('div.focal-point-chooser');
+
+  // To avoid an inline style, max-width and max-height are passed as data attributes
+  // from the focal-point-chooser class because they are dynamic values
+  $chooser.css('max-height', $chooser.data('max-height'));
+  $chooser.css('max-width', $chooser.data('max-width'));
```

Comment:
The original code directly sets the inline styles in the HTML and does nothing with it in the JavaScript. This means we only need to set it once on initial load.

So, I think we can move this to the on load function (the `$(function() {...})` below) so that it's only run once instead of also on resize and "remove" button click.